data/json/npcs/other/TALK_ANIMAL_SHELTER_SURVIVOR.json

```diff
@@ -0,0 +1,145 @@
+        "failure": { "topic": "TALK_DONE", "effect": "hostile" },
+        "condition": { "not": { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" } }
+      },
+      {
+        "text": "Hey, nice to meet you.",
+        "topic": "TALK_NPC_ANIMAL_SHELTER_SURVIVOR_INTRO",
+        "condition": { "not": { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" } }
+      },
+      {
+        "text": "<greet>",
+        "topic": "TALK_NPC_ANIMAL_SHELTER_SURVIVOR_INTRO",
+        "condition": { "npc_has_var": "knows_u", "type": "dialogue", "context": "first_meeting", "value": "yes" }
+      },
+      { "text": "Bye.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NPC_ANIMAL_SHELTER_SURVIVOR_INTRO",
+    "dynamic_line": "What can I do for you?",
+    "responses": [
+      { "text": "Who are you, exactly?", "topic": "TALK_NPC_ANIMAL_SHELTER_SURVIVOR_WHO" },
+      {
+        "text": "Wanna buy something, maybe sell off what you don't need?",
+        "topic": "TALK_NPC_ANIMAL_SHELTER_SURVIVOR_INTRO",
+        "effect": "start_trade"
+      },
+      {
+        "text": "Why don't you come with me?",
+        "trial": { "type": "PERSUADE", "difficulty": 15, "mod": [ [ "value", 2 ] ] },
+        "success": { "topic": "TALK_AGREE_FOLLOW", "effect": "follow", "opinion": { "trust": 1, "value": 1 } },
+        "failure": { "topic": "TALK_DENY_FOLLOW", "effect": "deny_follow", "opinion": { "trust": 0, "fear": 0 } }
+      },
+      { "text": "Could I buy one of your pets?", "topic": "TALK_NPC_ANIMAL_SHELTER_SURVIVOR_BUY_PETS" },
+      {
+        "text": "Any jobs you need done?",
+        "condition": { "and": [ { "not": "has_assigned_mission" }, { "not": "has_many_assigned_missions" } ] },
+        "topic": "TALK_MISSION_LIST"
+      },
+      { "text": "I gotta go.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "type": "talk_topic",
+    "id": "TALK_NPC_ANIMAL_SHELTER_SURVIVOR_WHO",
+    "dynamic_line": "I worked here before <the_cataclysm> dumped everything in the <swear> bin.  I took care of all the animals here, and when the <zombies> started roaming the streets, I gathered every pet or stray I could.  I still take care of them, just gotta worry about the <zombies> now on top of disease, cleanliness, etc.",
+    "responses": [
+      {
+        "text": "That's quite noble of you, but how do you take care of so many animals?",
+        "topic": "TALK_NPC_ANIMAL_SHELTER_SURVIVOR_PETS"
+      },
+      {
+        "text": "Wanna buy something, maybe sell off what you don't need?",
+        "topic": "TALK_NPC_ANIMAL_SHELTER_SURVIVOR_INTRO",
+        "effect": "start_trade"
+      },
+      {
+        "text": "Why don't you come with me?",
+        "trial": { "type": "PERSUADE", "difficulty": -15, "mod": [ [ "value", 2 ] ] },
+        "success": { "topic": "TALK_AGREE_FOLLOW", "effect": "follow", "opinion": { "trust": 1, "value": 1 } },
```

Comment:
If they've worked this hard to stay and keep the animals safe so far what's going to convince them to go along with you and potentially leave the animals behind?

Reply:
Nothing, you can't recruit them anymore.